design/token.go

```diff
@@ -198,6 +198,21 @@ var _ = a.Resource("token", func() {
 		a.Response(d.InternalServerError, JSONAPIErrors)
 		a.Response(d.Unauthorized, JSONAPIErrors)
 	})
+
+	a.Action("RevokeAll", func() {
+		a.Routing(
+			a.GET("/revoke_all"),
```

Comment:
Maybe `DELETE /users/:identityID/tokens` instead?

Reply:
That makes sense, changed.